lecture2gether-vue/src/App.vue

```diff
@@ -33,6 +33,7 @@ export default class App extends Vue {
 <style lang="scss">
     @use "src/styles/global_style";
     @import '~@mdi/font/css/materialdesignicons.css';
+    @import '../node_modules/typeface-roboto/index.css';
```

Comment:
Please use an absolute import like the line above

Reply:
Should be fine now